js/utils/Socket.js

```diff
@@ -15,11 +15,6 @@ function Socket(url) {
 Socket.prototype.connect = function(url) {
   var self = this;
 
-  // if already connected, do nothing
```

Comment:
Here, I'm reverting a change I made earlier. The idea was to avoid duplicate socket connections I thought I was seeing, but this fix resulted in the client not being able to reconnect to the same server after a lost connection.

I'm not able to regularly reproduce the duplicate socket issue, so I'm putting that aside from now.